x-pack/plugins/security_solution/cypress/integration/hosts/host_risk_tab.spec.ts

```diff
@@ -29,15 +34,33 @@ describe('risk tab', () => {
   });
 
   it('renders the table', () => {
+    kqlSearch('host.name: "siem-kibana" {enter}');
     cy.get(HOST_BY_RISK_TABLE_CELL).eq(3).should('have.text', 'siem-kibana');
     cy.get(HOST_BY_RISK_TABLE_CELL).eq(4).should('have.text', '21.00');
     cy.get(HOST_BY_RISK_TABLE_CELL).eq(5).should('have.text', 'Low');
+    clearSearchBar();
   });
 
   it('filters the table', () => {
     cy.get(HOST_BY_RISK_TABLE_FILTER).click();
     cy.get(HOST_BY_RISK_TABLE_FILTER_CRITICAL).click();
 
     cy.get(HOST_BY_RISK_TABLE_CELL).eq(3).should('not.have.text', 'siem-kibana');
+
+    // remove filter
+    cy.get(HOST_BY_RISK_TABLE_FILTER_CRITICAL).click();
```

Comment:
NIT: Wrap this method in a task one. If you create a method called "removeFilter" then you don't need to add the comment